### ISSUE TITLE
Difference in Result

### ISSUE THREAD

**Issue by NaeemKhan333** [swathikirans/GSM#6]
Thanks for nice work . I have trained a model on my own dataset which have three classes and every class have 20 videos each . I have formated the dataset in the somehting-something-v1 format and start the training. Durning training I have got testing accuracy as follow 

![ss1](https://user-images.githubusercontent.com/52487215/75686517-33c03e80-5cbe-11ea-9d80-1ffed88bc643.png)

After completing the training, I have tested the model on the same data, it giving me as follow

![Screenshot from 2020-03-02 19-41-05](https://user-images.githubusercontent.com/52487215/75686710-83066f00-5cbe-11ea-964a-1241bed7692d.png)

Class Accuracy 36.67%
Overall Prec@1 36.67% Prec@5 100.00%

can you explain the result ,is it ok or something wrong ?

**Comment by swathikirans** [comment]
Please make sure you are using the right model during inference. It looks like you are loading the something-v1 trained model during inference.

**Comment by NaeemKhan333** [comment]
Yes this is my model,which is trained using the something-something-v1 dataset, Al thought model name is ""something-v1_BNInception_avg_segment8_checkpoint.pth.tar"" but actully it is the model which is trained my me on my own dataset.
